tests/lca/imagebasedinstall/mgmt/deploy/tests/e2e-deploy-test.go

```diff
@@ -305,6 +336,128 @@ func createIBIOResouces(addressFamily string) {
+	createSharedResources()
+
+	By("Find cluster template configmap")
+
+	clusterTemplateConfigmap, err := configmap.ListInAllNamespaces(APIClient, metav1.ListOptions{
+		FieldSelector: fmt.Sprintf("metadata.name=%s", ibiClusterTemplateName),
+	})
+	Expect(err).To(BeNil(), "error encountered when listing configmaps mactching cluster template name")
+	Expect(len(clusterTemplateConfigmap)).To(Equal(1),
+		"error: received unexpected configmap count: %d", len(clusterTemplateConfigmap))
+
+	By("Find node template configmap")
+
+	nodeTemplateConfigmap, err := configmap.ListInAllNamespaces(APIClient, metav1.ListOptions{
+		FieldSelector: fmt.Sprintf("metadata.name=%s", ibiNodeTemplateName),
+	})
+	Expect(err).To(BeNil(), "error encountered when listing configmaps mactching node template name")
+	Expect(len(nodeTemplateConfigmap)).To(Equal(1),
+		"error: received unexpected configmap count: %d", len(nodeTemplateConfigmap))
+
+	clusterInstanceBuilder := siteconfig.NewCIBuilder(
+		APIClient, MGMTConfig.Cluster.Info.ClusterName, MGMTConfig.Cluster.Info.ClusterName).
+		WithPullSecretRef(MGMTConfig.Cluster.Info.ClusterName).
+		WithClusterTemplateRef(ibiClusterTemplateName, clusterTemplateConfigmap[0].Object.Namespace).
+		WithBaseDomain(MGMTConfig.Cluster.Info.BaseDomain).
+		WithClusterImageSetRef(ibiImageSetName).
+		WithClusterName(MGMTConfig.Cluster.Info.ClusterName)
+
+	if addressFamily == ipv4AddrFamily {
+		clusterInstanceBuilder.WithMachineNetwork(MGMTConfig.Cluster.Info.MachineCIDR.IPv4)
+	} else {
+		clusterInstanceBuilder.WithMachineNetwork(MGMTConfig.Cluster.Info.MachineCIDR.IPv6)
+	}
+
+	if MGMTConfig.PublicSSHKey != "" {
+		clusterInstanceBuilder.WithSSHPubKey(MGMTConfig.PublicSSHKey)
+	}
+
+	if MGMTConfig.ExtraManifests {
+		clusterInstanceBuilder.WithExtraManifests(extraManifestNamespaceConfigmapName).
+			WithExtraManifests(extraManifestConfigmapConfigmapName)
+	}
+
+	if MGMTConfig.SeedClusterInfo.Proxy.HTTPProxy != "" || MGMTConfig.SeedClusterInfo.Proxy.HTTPSProxy != "" {
+		clusterInstanceBuilder.WithProxy(&v1beta1.Proxy{
+			HTTPProxy:  MGMTConfig.SeedClusterInfo.Proxy.HTTPProxy,
+			HTTPSProxy: MGMTConfig.SeedClusterInfo.Proxy.HTTPSProxy,
+			NoProxy:    MGMTConfig.SeedClusterInfo.Proxy.NOProxy,
+		})
+	}
+
+	Expect(len(MGMTConfig.Cluster.Info.Hosts)).To(Equal(1), "error: can only support SNO deployments")
+
+	for host, info := range MGMTConfig.Cluster.Info.Hosts {
+		var bmcAddress string
+
+		if addressFamily == ipv4AddrFamily {
+			bmcAddress = info.BMC.URLv4
+		} else {
+			bmcAddress = info.BMC.URLv6
+		}
+
+		By("Add node entry for " + host)
+
+		siteconfigNode := siteconfig.NewNodeBuilder(host, bmcAddress, info.BMC.MACAddress, host, ibiNodeTemplateName,
+			nodeTemplateConfigmap[0].Object.Namespace).WithAutomatedCleaningMode("disabled")
+
+		if MGMTConfig.StaticNetworking {
+			if len(info.Network.Interfaces) != 1 {
+				Skip("Cannot support nodes with more than one network interface")
```

Comment:
why not?

Reply:
Just not something we have implemented yet